### PR TITLE
Fix function and sample test errors

### PR DESCRIPTION
**Pull request by diondrapeck**

### CODE REVIEW

File: R/model.R [hunk 1]
```diff
@@ -86,11 +86,11 @@ get_model <- function(workspace,
 register_model <- function(workspace,
                            model_path,
                            model_name,
+                           datasets,
```
Comment (heemanshusuri) [inline R/model.R]: could you move up the documentation of datasets after model_name?

Comment (diondrapeck) [reply R/model.R]: yep!

Comment (heemanshusuri) [inline R/model.R]: this should be an optional parameter, right?

Comment (diondrapeck) [reply R/model.R]: Good catch - thanks.

**Pull request merged**